sql/maintenance_utils.sql

```diff
@@ -39,11 +39,24 @@ CREATE OR REPLACE FUNCTION @extschema@.compress_chunk(
     hypercore_use_access_method BOOL = NULL
 ) RETURNS REGCLASS AS '@MODULE_PATHNAME@', 'ts_compress_chunk' LANGUAGE C VOLATILE;
 
+-- Alias for compress_chunk above.
+CREATE OR REPLACE PROCEDURE @extschema@.convert_to_columnstore(
+    uncompressed_chunk REGCLASS,
```

Comment:
Again, this parameter name doesn't really make much sense. Still it makes more sense here than in decompress_chunk.

```suggestion
    chunk REGCLASS,
```